tests/tests/control_c384gdas_wav

```diff
@@ -0,0 +1,132 @@
+                   RESTART/fv_core.res.tile2.nc \
+                   RESTART/fv_core.res.tile3.nc \
+                   RESTART/fv_core.res.tile4.nc \
+                   RESTART/fv_core.res.tile5.nc \
+                   RESTART/fv_core.res.tile6.nc \
+                   RESTART/fv_srf_wnd.res.tile1.nc \
+                   RESTART/fv_srf_wnd.res.tile2.nc \
+                   RESTART/fv_srf_wnd.res.tile3.nc \
+                   RESTART/fv_srf_wnd.res.tile4.nc \
+                   RESTART/fv_srf_wnd.res.tile5.nc \
+                   RESTART/fv_srf_wnd.res.tile6.nc \
+                   RESTART/fv_tracer.res.tile1.nc \
+                   RESTART/fv_tracer.res.tile2.nc \
+                   RESTART/fv_tracer.res.tile3.nc \
+                   RESTART/fv_tracer.res.tile4.nc \
+                   RESTART/fv_tracer.res.tile5.nc \
+                   RESTART/fv_tracer.res.tile6.nc \
+                   RESTART/phy_data.tile1.nc \
+                   RESTART/phy_data.tile2.nc \
+                   RESTART/phy_data.tile3.nc \
+                   RESTART/phy_data.tile4.nc \
+                   RESTART/phy_data.tile5.nc \
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
+                   RESTART/sfc_data.tile2.nc \
+                   RESTART/sfc_data.tile3.nc \
+                   RESTART/sfc_data.tile4.nc \
+                   RESTART/sfc_data.tile5.nc \
+                   RESTART/sfc_data.tile6.nc \
+                   out_pnt.points \
+                   out_grd.gnh_10m \
+                   out_grd.gsh_15m \
+                   out_grd.aoc_9km"
+
+export_fv3
+
+export TASKS=480
+if [[ $MACHINE_ID = cheyenne.* ]]; then
+  export TPN=36
+elif [[ $MACHINE_ID = hera.* ]]; then
+  export TPN=20
+  export THRD=2
+elif [[ $MACHINE_ID = wcoss_cray ]]; then
+  export TPN=6
+else
+  export TPN=12
+fi
+
+
+export INPES=6
+export JNPES=8
+export WRITE_GROUP=1
+export WRTTASK_PER_GROUP=24
+export FHMAX=12
+export NPZ=127
+export NPZP=128
+export NPX=385
+export NPY=385
+export DT_ATMOS=225
+
+export SYEAR=2021
+export SMONTH=03
+export SDAY=21
+export SHOUR=18
+export FHROT=3
+export FDIAG=3
+export NFHMAX_HF=0
+export IAU_OFFSET=6
+export CDMBWD=${CDMBWD_c384}
+export OUTPUT_GRID='gaussian_grid'
+export WRITE_DOPOST=.true.
+export IMO=1536
+export JMO=768
+
+export WARM_START=.true.
+export NGGPS_IC=.false.
+export EXTERNAL_IC=.false.
+export MAKE_NH=.false.
+export MOUNTAIN=.true.
+export NA_INIT=0
+export IAU_INC_FILES="'fv_increment3.nc','fv_increment6.nc','fv_increment9.nc'"
+export MODEL_INITIALIZATION=true
+export WLCLK=30
+
+export FNALBC="'global_snowfree_albedo.bosu.t766.1536.768.rg.grb',"
+export FNVETC="'global_vegtype.igbp.t766.1536.768.rg.grb',"
+export FNSOTC="'global_soiltype.statsgo.t766.1536.768.rg.grb',"
+export FNABSC="'global_mxsnoalb.uariz.t766.1536.768.rg.grb',"
+
+export CPL=.true.
+export CPLWAV=.true.
+export NEMS_CONFIGURE="nems.configure.leapfrog_atm_wav.IN"
+export CPLWAV=.T.
```

Comment:
@DeniseWorthen can I remove this? 
`export CPLWAV=.T. `
`as export CPLWAV=.true.` is there

Reply:
yes, you only need it once. The preference going forward is to use the full word .true. rather than the T/F.

Reply:
CPLWAV is still there twice.

Reply:
We investigated it before, removing either of them will break the test. so we kept it.